test/e2e/setup.go

```diff
@@ -377,9 +391,35 @@ var _ = BeforeSuite(func() {
 
 	if err := setup(context.Background()); err != nil {
 		panic(err)
+	} else {
+		m := statsd.New(
+			context.Background(),
+			log.WithField("component", "e2emetrics"),
+			_env,
+			os.Getenv("MDM_E2E_ACCOUNT"),
+			os.Getenv("MDM_E2E_NAMESPACE"),
+			os.Getenv("MDM_E2E_STATSD_SOCKET"),
+		)
+		e2eHook := e2elogging.NewE2EHook(m)
+		log.Logger.AddHook(&e2eHook)
 	}
 })
 
 var _ = AfterSuite(func() {
 	log.Info("AfterSuite")
 })
+
+var _ = JustAfterEach(func() {
+	csr := CurrentSpecReport()
+
+	metricName := strings.Replace(csr.ContainerHierarchyTexts[1], " ", ".", -1)
+
+	log.WithFields(logrus.Fields{
+		e2emetrics.LogEntryIsE2EEmittableMetric: "true",
```

Comment:
I would rather have the bool `true` here instead

Reply:
Now using bool `true` instead of string.

Reply:
I haven't been able to find any explicit policy saying that, however, when possible it is better to use flags that represent states, instead of binary flags because they can more easily be extended.